docs/demos/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Demos
+nav_order: 350
+description: Demos
+---
+
+<!--
+Copyright (C) 2016 and later: Unicode, Inc. and others.	Copyright (C) 2016 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html	License & terms of use: http://www.unicode.org/copyright.html
```

Comment:
In several files -- looks like all .md files and .json? -- the copyright parts are duplicated on each line, like here. Would you mind fixing these?

While you are at it, consider using "©" instead of "Copyright (C)".

Reply:
Done. Yikes, not sure how I managed to do that. PTAL.

Reply:
"Diff Across Force-Push" lgtm tnx!